sdk/communication/Azure.Communication.Chat/tests/samples/Sample3_ParticipantOperations.cs

```diff
@@ -33,7 +33,7 @@ public async Task GetAddRemoveMembersAsync()
             var chatParticipant = new ChatParticipant(josh)
             {
                 DisplayName = "Josh",
-                ShareHistoryTime = DateTime.MinValue
+                ShareHistoryTime = DateTimeOffset.MinValue
```

Comment:
Was this the only place where we use DateTime.Min (or Max) ?

Reply:
yes, that's right